modules/migration-setting-up-target-cluster-to-accept-source-dns-domain.adoc

```diff
@@ -15,12 +16,12 @@ For both non-secure HTTP access and secure HTTPS access, perform the following s
 +
 [source,terminal]
 ----
-$ oc expose svc app1-svc --hostname app1.apps.ocp3.example.com \
+$ oc expose svc app1-svc --hostname app1.apps.source.example.com \
  -n app1-namespace
 ----
 +
 With this new route in place, the server accepts any request for that FQDN and sends it to the corresponding application pods.
-In addition, when you migrate the application, another route is created in the target cluster domain (`app1.apps.ocp4.example.com`). Requests can reach the migrated application using both of these hostnames.
+In addition, when you migrate the application, another route is created in the target cluster domain (`app1.apps.target.example.com`). Requests can reach the migrated application using both of these hostnames.
```

Comment:
I think "either hostname" would be better than "both ...".

Change "can reach" to "reach". We have to be careful about using "can" because it has too many meanings.